protocol/development-workflow/README.md

```diff
@@ -1,171 +1,75 @@
 # Development Workflow
 
-- [Setting Up the Release Branch](#setting-up-the-release-branch)
```

Comment:
I think included a table of contents is still useful, so you can just click to jump to the role you are undertaking.